app.js

```diff
@@ -28,12 +28,11 @@ io.on('connection', (socket) => {
 
 	//When client requests for setting username
 	socket.on('set username', (name) => {
-		name = name.trim()
+		name = (name || "").trim()
 
 		//if name is empty(null), do nothing
-		if(name == "" || name == null) {
-			return;
-		}
+		if(!name) return socket.emit('user invalid', {username: `${name} is invalid.`});
```

Comment:
If you are adding this here then you have to add this socket in `client.js` too.

Reply:
Sorry, I forgot.

Reply:
Added the new event in the client. 4dc36b6